provider/qsd.py

```diff
@@ -292,6 +292,8 @@ def _fulfil_image_props(self, name, params):
             if params.get("force_create_image") == "yes" or params.get(
                     "create_image") == "yes":
                 image_created = True
+        if params.get("image_name") != name:
+            image_created = True
```

Comment:
hi @aliang123 , I am afraid here image_name and name are different concepts
for name, it's a tag defined in images, e.g. images = image1, then we can define params for image1, like
image_name_image1 = path/to/image1
so here you can see that image_name could be a subpath of 'image1'

Reply:
Yes, you're right, zhenchao. Have updated this filter to:
  if params.get("force_create_image", "yes") == "yes":
                LOG_JOB.info("QSD ready to create image %s", name)
                obj = qemu_storage.QemuImg(params, data_dir.get_data_dir(), name)
                _, result = obj.create(params)
                if result.exit_status != 0:
                    raise QsdError("Failed create image %s " % name)
                img["image_object"] = obj


And also in qsd_live_snapshot.cfg, we update force_image_create_stg1 to "no" when start qsd with sn1, then we can just replace the image_name, and don't need to create image again when we call self._fulfil_image_props(img, params)